econia-labs/emojicoin-dot-fun
# [ECO-2164] Add indexer WAF, REST API caching/auth key

# Description

1. Add header-based REST API key authentication.
1. Add REST API caching.
1. Update README with REST API key instructions.
1. Add a web application firewall (WAF) with assorted managed policies.
1. Add custom WAF rules for REST and WS endpoints, but leave disabled.

# Testing

To verify changes in this PR:
1. Follow the README for the `ECO-2164` stack to see `curl` results with and
   without header.
1. See the Web ACL traffic overview dashboards as described in the README.

# Checklist
